### ISSUE TITLE
Don't preview files that I open directly

### ISSUE THREAD

**Issue by JoshuaLamusga** [microsoft/vscode#70719]
The way that previewing files works today isn't ideal in my opinion. There's an option to disable file previewing, but I like having preview in some situations. I think it's a design error to open a file in preview mode if the user searches and opens it via Ctrl + P commands or directly clicking it in the file explorer. Files should be opened in preview mode only when they're navigated to automatically, by F12 to follow definitions/implementations, viewing results in the Search pane (Ctrl + Shift + F), and hitting breakpoints.

The bottom line is that in no circumstance should a file open in preview mode when the user directly opens it. I understand a number of people are already used to their current settings, but I think it would be beneficial to add a new previewing mode that works in the way I've described: respecting the user's choice to open a file directly, and previewing only files opened indirectly.

What does everyone think?

**Comment by ProLoser** [comment]
I don't agree, this is non-standard and I frequently open files trying to figure out if I need them. A preview becomes active when you start editing automatically, which is a clearer way to discern user intent. I think a side-effect of your scenario would actually create an unforseen issue: lots of dangling files open you didn't ACTUALLY care about.

**Comment by JoshuaLamusga** [comment]
To clarify my request, it's based on my workflow. I tend to open files I need a lot more often than ones I don't, closing tabs actively when done. If I have a non-standard workflow, I'm curious to know how many others might benefit from this suggestion so it can be weighed whether it's worth supporting different preview behavior: keeping on edit vs. on open (with the exceptions in the original post above). I'm guessing my workflow is too specific to support, in this case.
